src/runtime_src/core/common/xclbin_parser.h

```diff
@@ -97,6 +97,23 @@ struct softkernel_object
   char *sk_buf;
 };
 
+// struct aieresourcesbin_object - wrapper for a aie resources object
+//
+// @symbol_name: aie resources symbol name
+// @size: size of aie resources image
+// @start_col: start column
+// @num_col: number of columns
+// @ar_buf: pointer to the aie resourecs buffer
+struct aieresourcesbin_object
```

Comment:
We shouldnt require this in userspace. xclbin.h should already have resources information